src/breadcrumbs/breadcrumbs.macros.html

```diff
@@ -24,21 +23,25 @@
         </a>
       </li>
       <li class="breadcrumbs-list-item">
-        {{ icon('arrow', size='tiny') }}
+        >
       </li>
     {% endif %}
 
     {% for item in data %}
       {% if loop.first %}
         <li class="breadcrumbs-list-item">
-          {{ link(item.content, href=item.href, class='breadcrumbs-link') }}
+          <a class="breadcrumbs-link" href="item.href">
+            {{ item.content }}
+          </a>
         </li>
       {% else %}
         <li class="breadcrumbs-list-item">
-          {{ icon('arrow', size='tiny') }}
+          >
         </li>
```

Comment:
Why do we want to wrap the list separator in an `li`, it isn't part of the list. A span within the li or maybe pseudo before/after elements could work here.